Trell.Engine/ClearScriptWrappers/EngineWrapper.cs

```diff
@@ -141,6 +141,29 @@ DocumentContextCallback contextCallback
         return result;
     }
 
+    internal void LoadJsLibrary(string fullFilePath, DocumentCategory category) {
```

Comment:
This may need to become a more formal thing at some point to support `IPlugin` but is great for now.